src/Dialog/Dialog.test.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import {fireEvent, render, waitFor} from '@testing-library/react'
```

Comment:
Curious to hear if there is a reason to use fireEvent here instead of userEvent 👀 I have been using userEvent mostly based on what I learnt from the differences from fireEvent ([docs reference](https://testing-library.com/docs/user-event/intro/#differences-from-fireevent)) but curious to hear what you think 🙌🏻

Reply:
I just copied some stuff over from the old Dialog tests. I'll update to use `userEvent` instead 🙂